test/test-index-column.rb

```diff
@@ -63,6 +63,15 @@ def test_data?
         not @index.data?
       end
     end
+
+    def test_lexicon?
+      assert do
+        @terms.lexicon?
+      end
+      assert do
+        not @index.lexicon?
+      end
+    end
```

Comment:
Could you split these assertions? One test should check one thing. If the first check is failed, the second check isn't ran. It reduces information to debug.

```ruby
sub_test_case("#lexicon?") do
  def test_true
     assert do
      @terms.lexicon?
    end
  end

  def test_false
    assert do
      not @index.lexicon?
    end
  end
end
```